File: flight_busiest_traveling_period/v1/get/Node SDK/flight_busiest_traveling_period.js
const WAConvo = require("waconvo");

const waconvo = new WAConvo({
  clientId: "YOUR_API_KEY",
  clientSecret: "YOUR_API_SECRET",
});

async function main() {
  try {
    // What were the busiest months for Madrid in 2017?
    const response =
      await waconvo.travel.analytics.airTraffic.busiestPeriod.get({
        cityCode: "MAD",
        period: "2017",
        direction: WAConvo.direction.arriving,
      });

    console.log(response);
  } catch (error) {
    console.error(error);
  }
}

main();

File: flight_busiest_traveling_period/v1/get/Python SDK/flight_busiest_traveling_period.py
# Install the Python library from https://pypi.org/project/waconvo
from waconvo import Client, ResponseError

waconvo = Client(
    client_id='YOUR_WACONVO_API_KEY',
    client_secret='YOUR_WACONVO_API_SECRET'
)

try:
    '''
    What were the busiest months for Madrid in 2022?
    '''
    response = waconvo.travel.analytics.air_traffic.busiest_period.get(
        cityCode='MAD', period='2017', direction='ARRIVING')
    print(response.data)
except ResponseError as error:
    raise error
